lib/encoding/json.toit

```diff
@@ -52,14 +52,14 @@ The $converter block is passed an object to be serialized and an instance
   of the $Encoder class.  If it returns a non-null value, that value will
   be serialized instead of the object that was passed in.  Alternatively,
   the $converter block can call the $Encoder.encode, $Encoder.put-list,
-  or Encoder.put-unquoted methods on the encoder.
+  or $Encoder.put-unquoted methods on the encoder.
 */
 encode-stream --writer/io.Writer obj [converter] -> none:
```

Comment:
Could be done in a separate PR or here, but just `encode o --writer` and `decode --reader` would actually be nicer than the `-stream` suffix.
Would require to deprecate the existing `decode-stream` method, so maybe a follow-up PR is better.